default_warehouse_from_sale_team/models/default_warehouse_mixin.py

```diff
@@ -47,7 +47,9 @@ def create(self, vals):
 
     def _get_salesteam_from_vals(self, vals):
         """Determine sales team from creation values"""
-        if "name" in vals:
+        name_field = self._fields["name"]
+        default_name = name_field.default(self) if callable(name_field.default) else name_field.default
+        if vals.get("name", default_name) != default_name:
```

Comment:
This doesn't seem correct to me. Why is this necessary?

Reply:
As discussed earlier I'll close this PR since [this new PR](https://github.com/Vauxoo/addons-vauxoo/pull/1664) solve the issue of why this was created.